### ISSUE TITLE
Add support to log_file option

### ISSUE THREAD

**Issue by pplucky** [danielwelch/hassio-zigbee2mqtt#311]
From [zigbee2mqtt](https://www.zigbee2mqtt.io/information/configuration.html):
```
  # Optional: Log file name, can also contain timestamp, e.g.: zigbee2mqtt_%TIMESTAMP%.log (default: shown below)
  log_file: log.txt
```

**zigbee2mqtt add-on version (if edge, please report commit hash):**
1.12.0

**Operating environment (HassOS, Virtual Machine, Device/platform running Hass.io):**
Hass.io on Docker, over Raspbian

**Description of problem:**
log_file option not working

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```json
data_path: /share/zigbee2mqtt
devices: devices.yaml
groups: groups.yaml
homeassistant: false
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://core-mosquitto:1883'
  user: *** redacted ***
  password: *** redacted ***
serial:
  port: /dev/ttyACM0
  disable_led: true
advanced:
  availability_timeout: 60
  log_level: info
  log_directory: /share/zigbee2mqtt/log
  log_file: log_%TIMESTAMP%.txt
  last_seen: ISO_8601
  pan_id: 6754
  channel: 11
  network_key:
    - 1
    - 3
    - 5
    - 7
    - 9
    - 11
    - 13
    - 15
    - 0
    - 2
    - 4
    - 6
    - 8
    - 10
    - 12
    - 13
  availability_blacklist: []
ban: []
whitelist: []
queue: {}
socat:
  enabled: false
  master: 'pty,raw,echo=0,link=/dev/ttyZ2M,mode=777'
  slave: 'tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5'
  restartdelay: 1
  initialdelay: 1
  options: '-d -d'
  log: false
```

**Your logs from Hass.io**
`20-03-19 10:49:29 WARNING (MainThread) [supervisor.addons.validate] Unknown options log_file`

**Comment by pplucky** [comment]
No luck in getting this implemented?

**Comment by pplucky** [comment]
No luck in getting this implemented?

**Comment by ciotlosm** [comment]
Fixed in #353
